src/main/steps/c100-rebuild/safety-concerns/abduction/content.ts

```diff
@@ -1,21 +1,21 @@
 import { TranslationFn } from '../../../../app/controller/GetController';
 
 const en = () => ({
-  guidanceOnAbductionLabel: 'Guidance on parental child abduction',
-  guidanceOnAbductionLink: 'https://www.gov.uk/government/collections/child-abduction',
-  getHelpOnReturningChildAbroadLabel: 'Get help to return a child from abroad or arrange contact',
-  getHelponReturningChildAbroadLink: 'https://www.gov.uk/return-or-contact-abducted-child',
-  preventChildFromObtainingPassportLabel: 'Stop a child from getting a passport',
-  preventChildFromObtainingPassportLink: 'https://www.gov.uk/stop-child-passport',
+  guidanceParentalChildHyperlink: 'https://www.gov.uk/government/collections/child-abduction',
```

Comment:
Did you rename al these label & link field names ?

Reply:
fixed